src/main/java/com/oracle/nosql/spring/data/core/NosqlTemplateBase.java

```diff
@@ -178,6 +186,151 @@ protected boolean doCreateTableIfNotExists(
         return tableState == TableResult.State.ACTIVE;
     }
 
+    protected void doCheckExistingTable(NosqlEntityInformation<?, ?> entityInformation) {
+        final String columnsField = "fields";
+        final String columnNameField = "name";
+        final String columnTypeField = "type";
+        final String shardField = "shardKey";
+        final String primaryField = "primaryKey";
+        final String ttlField = "ttl";
+        final String identityField = "identity";
+
+        TableResult tableResult = doGetTable(entityInformation);
+        /*If table already exist in the database compare and throw error if
```

Comment:
Comments should have spaces between comment start/end and words: For example:
/* comment */
// comment

Reply:
Is it a standard? Is it part of checkstyle?

Reply:
This is Java recommended way: https://www.oracle.com/java/technologies/javase/codeconventions-comments.html
I don't think there is a checkstyle for this.

Reply:
Fixed